packages/ui-file-drop/babel.config.js

```diff
@@ -29,7 +29,9 @@ module.exports = {
       {
         coverage: Boolean(process.env.COVERAGE),
         esModules: Boolean(process.env.ES_MODULES),
-        removeConsole: process.env.NODE_ENV === 'production',
+        removeConsole:
+          process.env.NODE_ENV === 'production' &&
+          process.env.GITHUB_PULL_REQUEST_PREVIEW !== 'true',
```

Comment:
I think we can so this in a much easier way. You don't need to change any of the `babel.config.js` files, just `ui-babel-preset/lib/index.js` line 111:

```if (opts.removeConsole) {``` to

```if (process.env.GITHUB_PULL_REQUEST_PREVIEW !== 'true' && opts.removeConsole) {```

This should keep console.log in our preview builds